Library fails to compile on framework-arduinoespressif32 @ 3.0.0

I have some hardware in my project that I can only get working with Arduino V3.0.0, so I am using this alpha version.

__Steps to reproduce:__

1. add "tobiasschuerg/ESP8266 Influxdb @ ^3.13.1" to lib-deps section of platformio.ini
2. Try to compile

__Expected behavior:__
Compile the project

__Actual behavior:__
I get this error:
`Compiling .pio\build\esp32-s3-devkitc-1\libd2f\ESP8266 Influxdb\Point.cpp.o 
.pio/libdeps/esp32-s3-devkitc-1/ESP8266 Influxdb/src/HTTPService.cpp: In constructor 'HTTPService::HTTPService(ConnectionInfo*)':
.pio/libdeps/esp32-s3-devkitc-1/ESP8266 Influxdb/src/HTTPService.cpp:37:5: error: 'WiFiClientSecure' was not declared in this scope; did you mean 'WiFiClientRxBuffer'?
   37 |     WiFiClientSecure *wifiClientSec = new WiFiClientSecure;
      |     ^~~~~~~~~~~~~~~~
      |     WiFiClientRxBuffer
.pio/libdeps/esp32-s3-devkitc-1/ESP8266 Influxdb/src/HTTPService.cpp:37:23: error: 'wifiClientSec' was not declared in this scope; did you mean '_wifiClient'?
   37 |     WiFiClientSecure *wifiClientSec = new WiFiClientSecure;
      |                       ^~~~~~~~~~~~~
      |                       _wifiClient
.pio/libdeps/esp32-s3-devkitc-1/ESP8266 Influxdb/src/HTTPService.cpp:37:43: error: 'WiFiClientSecure' does not name a type; did you mean 'WiFiClientRxBuffer'?
   37 |     WiFiClientSecure *wifiClientSec = new WiFiClientSecure;
      |                                           ^~~~~~~~~~~~~~~~
      |                                           WiFiClientRxBuffer
*** [.pio\build\esp32-s3-devkitc-1\libd2f\ESP8266 Influxdb\HTTPService.cpp.o] Error 1`

__Specifications:__
 - Library Version: 3.13.1
 - InfluxDB Version: n/a
 - Board/ESP chip: ESP32-S3
 - Device Arduino SDK version: 3.0.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,4 +1,8 @@
 # Changelog
+## 3.13.2 [2024-06-04]
+### Fixes
+- [236](https://github.com/tobiasschuerg/InfluxDB-Client-for-Arduino/pull/236) - Fix compilation problem on ESP32 Core 3.0.0
+
 ## 3.13.1 [2023-03-08]
 ### Fixes
 - [210](https://github.com/tobiasschuerg/InfluxDB-Client-for-Arduino/pull/210) - Allow setting of options without previously set connection params

--- a/src/HTTPService.h
+++ b/src/HTTPService.h
@@ -33,6 +33,8 @@
 # include <ESP8266HTTPClient.h>
 #elif defined(ESP32)
 # include <HTTPClient.h>
+# include <WiFiClient.h>
+# include <WiFiClientSecure.h>
 #else
 # error "This library currently supports only ESP8266 and ESP32."
 #endif
@@ -132,3 +134,4 @@ public:
 };
 
 #endif //_HTTP_SERVICE_H_
+

